### ISSUE TITLE
Add Dark UI

### ISSUE THREAD

**Issue by ghost** [brave/browser-laptop#271]
As there are people around which are unable to look at a light UI only because of their eyes. I am such a person, but there are tons of others around with such an issue.

Hopefully you take this into account for future releases. Thanks!

----

edit by @luixxiul

If you have a serious eye problem, definitely f.lux is worth trying.
You can grab a copy here: https://justgetflux.com/

also see below: https://github.com/brave/browser-laptop/issues/271#issuecomment-336803894

> Until every variable which defines color of the UI will be moved to [`theme.js`](https://github.com/brave/browser-laptop/blob/master/app/renderer/components/styles/theme.js), it will __never__ be available.

**Comment by PFCKrutonium** [comment]
:+1:

**Comment by bnvk** [comment]
This would be great to have :+1: indeed!

**Comment by Adejair** [comment]
:+1:

**Comment by bbondy** [comment]
might be a little early to have it while we're building out the UI elements, but definitely would like to have this later. Ideally an easy way to install the themes in.

**Comment by D0J0P** [comment]
Brave has this nice Mac-like light UI, so having a dark Mac-like UI would be nice too, like you've seen on Final Cut Pro/Garageband and so on.

**Comment by Arandhras** [comment]
I like this, too! Please implement it!

**Comment by bradleyrichter** [comment]
coming to a future near you:

![image](https://cloud.githubusercontent.com/assets/13509546/15482284/a7c17bf2-20e3-11e6-971b-21444264af80.png)


Can't wait to design it. I'm currently using PSD in dark and AI in light because the toolbars are otherwise confusing!

**Comment by SJ-Diamond** [comment]
Hopefully this includes future elements too like history background, download/bookmark page background.

**Comment by wastedsunsets** [comment]
Dark theme would have this browser as my daily driver.

**Comment by idprism** [comment]
upvote. more likely to use applications that dont hurt my eyes on open.

**Comment by srirambv** [comment]
+1 from [support ](https://linkbubble.zendesk.com/agent/tickets/7150)

**Comment by yannduran** [comment]
👍

**Comment by ace72** [comment]
+1 dark theme as of now cant use this browser at night hurts my eye i hope this gets implemented real soon!

**Comment by pietercdevries** [comment]
Is this issue resolved yet? I love the browser but I'am forced to use chrome since I can install dark theme on browser and Gmail. I have a sensitivity to light and already have my displays brightness down. Thanks

**Comment by Liunkae** [comment]
@pietercdevries: Not yet, but it will be implemented before or with the release of Brave 1.0.

**Comment by pietercdevries** [comment]
Thank you very much!

**Comment by ninthbelief** [comment]
Just commenting to say that I'm interested as well. The lack of a dark theme is the number one reason I haven't switched to Brave yet.

**Comment by AnthoniG** [comment]
![image](https://cloud.githubusercontent.com/assets/11777453/24392959/07dfd7de-138e-11e7-8e0e-1593d6fbea0b.png)

Just to say that I too really want this.
I have a serious eye problem and can not use Brave as it's too light for me and seriously hurts my eyes.
Adding a Dark theme / UI options would be brilliant.

**Comment by luixxiul** [comment]
https://github.com/brave/browser-laptop/issues/7953#issuecomment-290164529

> Another workaround for macOS, Linux and Windows: install f.lux which can set a warm white tint in the evening which prevents a harsh white screen.

+1 from me too. If you have a serious eye problem, definitely the software is worth trying.

**Comment by srirambv** [comment]
+1 from community on allowing dark theme https://community.brave.com/t/gray-is-so-depressing/2091/2

**Comment by eljuno** [comment]
+1 [from community](https://community.brave.com/t/extension-and-black-brave/4779)

**Comment by Dudes1** [comment]
Looking forward to this too.

**Comment by JackDinn** [comment]
Me too, it will be my default browser if a dark UI can be archived. Please please cos i really like this browser and FF is dying a death v soon.

**Comment by bsclifton** [comment]
+1 from @savolla via https://github.com/brave/browser-laptop/issues/10065 😄

**Comment by Dudes1** [comment]
And for the Android:

https://github.com/brave/browser-android-tabs/issues/81

**Comment by ChillyHellion** [comment]
This will be a big plus in reaching feature parity with other major browsers.  So far Firefox, Chrome, Opera, and Vivaldi all support dark themes.  Brave kicks butt in the privacy and performance department, but I'm glad user interface improvements are being focused on as well.

**Comment by JonnyRedHed** [comment]
Dark UI mode will be great. But you could also go the extra mile and also include ALL settings, extensions etc pages in the dark mode UI once turned on.   Maybe even alow each settings page to override dark mode if one so desires.    Making it a full and proper dark mode, front UI and back end pages and UI dark mode.  Mirror over to Android as well please.

**Comment by AAGZ0921** [comment]
Yes, I can't wait, I want to ditch chrome for good, F.lux is good, but not enough.  hopefully this becomes a reality very soon.

**Comment by AnthoniG** [comment]
I want to ditch Firefox. Of late (and each time they release a new version) it's going so slow and getting really annoying.

Want Brave as my new browser but unfortunately it's just not there yet for me to use it full time.

**Comment by luixxiul** [comment]
dark ui is planned in 1.0. hang tight.

**Comment by ChillyHellion** [comment]
Excellent, thank you for the info!

**Comment by rtloeffler** [comment]
Waiting for the dark theme!!!!!!
Bonus points, Dark theme for android! :)

**Comment by JackDinn** [comment]
Ahh, I was getting excited when i see a Brave update today , i updated it with fingers crossed and .... booo ,

 "This is not the update you are looking for" .... I cant feel the force just yet :(

**Comment by Dudes1** [comment]
Thanks, Serg, for maintaining the app.

**Comment by egferro** [comment]
Please also allow to change theme and color,  grey really sucks. Other than than fantastic job!

**Comment by JonnyRedHed** [comment]
Dark UI mode will be great. But you could also go the extra mile and also include ALL settings, extensions etc pages in the dark mode UI once turned on.   Coming from a dark webpage made dark by either Stylus or 'Care your eyes' extension (chrome based, Opera etc) onto a bright white background settings page late at night just kills the night eyes.

Maybe even allow each settings page to override dark mode if one so desires.    Making it a full and proper dark mode, front UI and back end pages and UI dark mode.  Mirror over to Android as well please.   

Be the first dev team to have truly dark mode UI right through the browser on all platforms.

------------

**Care your eyes** - chrome extension - can be used in Opera, Chrome, Vivaldi etc.  Good for random web pages you want dark.  Uses a nice dark brown hue. Remembers top level domains.  

*set up: choose white list mode,  then just use the 'Enable at this site'  to make a page dark and have it remember.   Dev is working on remembering single pages rather than just top level domain.

https://chrome.google.com/webstore/search/care%20your%20eyes

--
**Stylus** - better than Stylish - no data mining etc of your history and habits.
https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne?hl=en

**Stylus Dark UI** - Manage styles page etc with the stylus UI in dark mode.
https://userstyles.org/styles/142097/stylus-gray-matter-chromium-based-or-firefox

**Comment by bradleyrichter** [comment]
"True-dark” is the MVP. No retina-popping white background-color allowed.   :   )

Our styles are almost ready for this feature so it has moved up in the list.

**Comment by rtloeffler** [comment]
@bradleyrichter is it going to the weekly build version? If so I'm downloading that right now!!

**Comment by bradleyrichter** [comment]
Not yet but I will pour on the pressure. Dark mode is now in the top 5 most requested list.

**Comment by JonnyRedHed** [comment]
Can I ask what the dev team think about extending the dark mode onto ALL the back end settings and extension/security pages.

**Comment by Liunkae** [comment]
Most context menus are native context menus, so we'd have to use the non-native (React) menus to add style to them. I know it's doable in most places, but I'm not sure about everywhere.

**Comment by JonnyRedHed** [comment]
I see.

For example, see this FF dark context menu style for Stylus/Stylish.  (your have to install it on FF to see it).

https://userstyles.org/styles/98802/dark-context-menu-for-firefox

**Comment by Liunkae** [comment]
I think Firefox is using non-native context menus. There's a subtle difference in the style and animation compared to native context menus, at least on my OS (macOS).

**Comment by JonnyRedHed** [comment]
I see, thanks for explaining.

**Comment by boldfilter** [comment]
Literally the only thing holding me back from using your browser over vivaldi or firefox

**Comment by AnthoniG** [comment]
I've dropped Firefox completely now and use PaleMoon instead (which is a branch of the Firefox code from earlier).

I now use Brave as my development tracking browser.
IE:- Github pushing, Bitbucket pull and push / tracking. e9.io etc, etc :grinning:

**Comment by JonnyRedHed** [comment]
So, are the dev team going to implement extending the dark mode onto ALL the back-end settings and extension/security pages etc.  Is this something you can do soon.  

Maybe also add some settings which show once dark mode is turned on.

**Comment by eljuno** [comment]
+1 from community https://community.brave.com/t/dark-theme-is-a-must-for-you-guys-it-is-really-critical/6726?u=eljuno

**Comment by Stabington** [comment]
+1 must-have feature.
Issue is over a year old now. Surely it can't be that difficult.

**Comment by hladf** [comment]
any news about dark theme?
why not make the dark a default theme??????????

**Comment by xywolap** [comment]
Great browser but far too bright.  Just an option for say a dark grey or similar background would help.

**Comment by nimnimnim** [comment]
Keep up the good work!

**Comment by A9G-Data-Droid** [comment]
I just looked through the settings expecting to find a dark option. I'm glad this thread is already rolling with steam. I use Brave all the time and I'm looking forward to the new theme!

**Comment by JackDinn** [comment]
This about the 3rd/4th update since i started waiting but still no dark theme, thought it was coming in the "next" update :-/ O well im not going anywhere.

**Comment by xywolap** [comment]
I found a fix... I deleted brave and installed firefox... all is good (perfect actually)

**Comment by JonnyRedHed** [comment]
Brave devs,  please just give us the dark UI and all settings/security/extensions pages as well.  Very important to also cover the back end pages for a true dark mode UI.    And then in time work on also allowing the on page right click context menu and all over drop down menus to have a dark mode option.   But settings for those, so one can turn dark mode off for right click and drop downs if default white backgrounds is preferred.  

Back end settings/security/extensions and flags or what ever can also have toggle switches at the top of each page so when the main dark mode UI is turned on they are active so you've the option to turn back to default white BG on some or all settings pages.

Setting for all.

**Comment by luixxiul** [comment]
Until every variable which define color on the UI will be moved to [`theme.js`](https://github.com/brave/browser-laptop/blob/master/app/renderer/components/styles/theme.js), it will __never__ be available.

@cezaraugusto and I am working on that. 

Here's a list of my PRs, some of which are for that task: https://github.com/brave/browser-laptop/issues?q=assignee%3Aluixxiul+is%3Aopen+label%3Arefactoring%2Faphrodite

I cannot promise the date when it'll be finished, but it will be finished _anyway_.

**Comment by luixxiul** [comment]
https://github.com/brave/browser-laptop/issues/271#issuecomment-324266684

> So, are the dev team going to implement extending the dark mode onto ALL the back-end settings and extension/security pages etc. Is this something you can do soon.

No, I don't think it will happen soon.

The 1st phase will only be limited only to UI. Otherwise it would take too much time to ship the dark theme.

The dark about pages will be considered after that.

**Comment by JonnyRedHed** [comment]
Maybe in the future of web browser and other software, the thought of dark mode UI's will be designed/baked in right from the concept phase. So it is easier to implement and fine tune.

Having a dark mode UI for the main browser to me also logically means all back end pages, add-ons/extensions, about, settings, security, shields, experiments/flags.    And also optional drop down menus and on page right click context menu option for having that dark as well.   All with settings so the user can customise fully or just a little.  

Dark mode UI in the true sense.

**Comment by luixxiul** [comment]
@JonnyRedHed you are always welcome to join us to refactor the code to implement the dark UI ⚡️

**Comment by eljuno** [comment]
+1 from community https://community.brave.com/t/win64-laptop-v0-19-95-general-feedback-minor-issues/11283?u=eljuno

**Comment by eighteyes** [comment]
I can't look at big patches of white. Until you add dark mode, or another extension to invert web site colors, I cannot use brave. I like consuming text, and I would pay writers via brave if I could consume text comfortably. But I cannot. 

This is a business decision. Take it upstairs! :)

**Comment by JonnyRedHed** [comment]
@luixxiul  I don't know code sorry to say.  But I do know 100% what a I want to see and use as and end user.

**Comment by eljuno** [comment]
+1 from community https://community.brave.com/t/please-for-the-love-of-some-deity-add-a-dark-theme/12174?u=eljuno

**Comment by GronbergInfo** [comment]
@bradleyrichter A 1.5 year later since your "comming soon" picture, no such option what i can find.

**Comment by JackDinn** [comment]
dont kid yourself they are never going to do a dark theme. just move on.

**Comment by A9G-Data-Droid** [comment]
No need for pessimism @JackDinn. What seems hard is sometimes easy. What seems easy is sometimes hard. It sounds like they coded the UI in a way that would make a simple change like color more difficult than it needs to be. The same thing happens with localization projects. If you don't code your entire project to have variable text from the beginning it can be very difficult to change every declared string to a variable and then point those variables to a selector that can pick translated strings. This is similar. They need setting that can change every color in the whole project. These colors may have been hardcoded instead of having variables.

**Comment by JackDinn** [comment]
they were hard coded which is why they will have a hell of a job to change it and they have much more important things to be doing which they will continue to have. Im not saying definitely never but i personally just dont see it. Plus in the next year or so who knows what other browser offerings there might be.

**Comment by JonnyRedHed** [comment]
Proper dark modes are just expected nowadays as we move into 2018, well I expect them anyway from a new browser like Brave.   I have dark everything, everywhere when possible.  Should be there as an option for everything in my opinion.  Please consider moving it back into the important features of your developments 2018 timeline.

**Comment by Vispocks** [comment]
bradleyrichter:  I don't see that in Adv Settings?

**Comment by Liunkae** [comment]
@Vispocks That is just a mockup. Dark mode has not been implemented yet.

Good news for everyone: Many of the tasks needed for dark mode are either completed or pending: https://github.com/brave/browser-laptop/projects/15

**Comment by VincentBurke** [comment]
+1 to dark mode, I'll switch from firefox happily once this is implemented!

**Comment by glennsteen** [comment]
No dark mode != me using brave. I really want to leave FF, so please....

**Comment by kj4ezj** [comment]
I would really just like Brave to import the Chrome extension Deluminate for this purpose. Deluminate is perfect!
https://chrome.google.com/webstore/detail/deluminate/iebboopaeangfpceklajfohhbpkkfiaa

**Comment by swiftaes** [comment]
+1 for dark mode, light sensitivity is lame

**Comment by bwydro** [comment]
been using Brave for the last 3 months or so - the dark mode is the last thing that is missing :-) Please make it happen.

**Comment by pdimri** [comment]
I like this browser a lot but waiting for dark mode to come so that I can make it my default browser. I am bit disappointed. Till then I am going to use Caf based browser for night mode.

**Comment by informationeagle** [comment]
Definitely implement a dark mode or theme but, in the meantime, f.lux did help... a lot.  I had glossed over it a while ago but didn't quite get it.  Now, thanks to the Brave Community, I am beginning to see how nicely done it is.  I just read that the dark mode is closer to launch, thanks for an excellent browser and keep up the great work!

**Comment by u147865** [comment]
dark mode is needed, ASAP. I hate light themes.

**Comment by pdimri** [comment]
Any timeline to get this feature ? I m using Caf based browser once brave
have this feature I can make it my default browser.

On Jan 10, 2018 at 9:57 PM, <u147865 <notifications@github.com>> wrote:

dark mode is needed, ASAP. I hate light themes.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/brave/browser-laptop/issues/271#issuecomment-356817668>,
or mute the thread

**Comment by Vispocks** [comment]
my own custom based chrome (black theme)

with extensions:
ScriptSafe
Ghostery
HTTPS Everywhere
AdBlock
Dark Mode (for every page with options)

<sorry!> brave

**Comment by midnitehack** [comment]
I am a 3 day brave groupie. Love the spirit behind the project. Its hot. But my eyes are dying researching and trading cryptos for hours on end on a 32inch monitor. The other groupies are not just talking about this dark theme. It is beyond critical. It is beyond needed. It is worth 1 satoshi a piece contribution from everyone here... Please lets get it this new year, 1st Q. Thanks for listening to your devoted bravers.

**Comment by JonnyRedHed** [comment]
Yes not just creating your own dark mode UI, but also some way to darken all your webpages you visit.  Either like the 'care your eyes' extension below. And or allow the Stylus extension as well so we can add page styles.

But until then...

**Care your eyes** - chrome extension - can be used in Opera, Chrome, Vivaldi etc.  Good for random web pages you want dark.  Uses a nice dark brown hue for the dark mode which is quite a nice balance of night time or dimmed room colors. Remembers top level domains.   Dev is working on remembering single pages rather than just top level domain.

*set up: choose white list mode,  then just use the 'Enable at this site'  to make a page dark and have it remember.     Very easy for those random web pages and articles you want dark.

https://chrome.google.com/webstore/search/care%20your%20eyes

--
**Stylus** - better than Stylish - no data mining etc of your history and habits.
https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne?hl=en

**Stylus Dark UI** - Manage styles page etc with the stylus UI in dark mode.
https://userstyles.org/styles/142097/stylus-gray-matter-chromium-based-or-firefox

**Comment by Monakiel** [comment]
Yes, I'm waiting for it too ! +1

**Comment by Gmike901** [comment]
MY EYESSS!!!

Tried to make the switch today, but can't live without dark UI / night mode (RES).

**Comment by MuchAtHome** [comment]
+1 ASAP ...

**Comment by JonnyRedHed** [comment]
Supporting Stylus, - chrome extension will be perfect for a more permanent website darken.  And and the 'Care your eyes' - chrome extension for just random white BG pages you want to just change in the moment.  

But,  if the Brave devs really wanted to take a big 'darker web page' leap forward they could take a good look and the 'Care your eyes' - chrome extension and integrate the best dark options into Brave itself. So one can choose from a few different styles of darker themes.  Have it remember not only the domain level but also just the webpage level.  With a remembered domains/pages settings backup option.  

Then one they do implement the first iteration of dark mode UI, they would also have an integrated on-page dark mode as well.  The first browser to have both dark UI and its own webpage darken feature.

Pretty interesting idea I think....  and also support Stylus :)

**Comment by adamlacombe** [comment]
👍

**Comment by ikimmybee** [comment]
2 Years later and still no dark UI. OTL

**Comment by fiatphos** [comment]
+1 for dark UI, please. 

May I suggest something like a color-inversion feature, if it can be implemented sooner? I use this extension (High Contrast - picture attached with this comment) in Chrome to invert the color of the pages. I do have to disable it occasionally on some pages - such as pages with lots of pictures - but for the most part I keep it enabled all the time and I am able to browse the web with much less strain to my eyes.

![image](https://user-images.githubusercontent.com/31344773/35495009-96e29726-04e4-11e8-8291-fca665259fe5.png)

**Comment by RobLoach** [comment]
@fiatphos This is about the User Interface being dark. For the content of the websites being dark, I'd suggest checking out [Stylish](https://userstyles.org/) over at https://github.com/brave/browser-laptop/issues/5727 .

**Comment by AAGZ0921** [comment]
Still waiting on this.  Been too long.  I get updates almost daily, nothing, zip, zero, nada.  I'm just going to move back to firefox.

**Comment by pdimri** [comment]
New update landed on my phone but nothing related to dark mode. Please implement something like Samsung browser's dark/ high contrast mode. I have high expectations from brave team .

**Comment by souziago** [comment]
Migrating from Firefox... No Dark mode... Migrating to Firefox...

**Comment by AnthoniG** [comment]
I've finally given up on this. I am now using Palemoon which is a branch of Firefox **before** they removed the extension API

**Comment by jamesray1** [comment]
Would be really good to get this in my dark room where I have to squint with the light on anyway, day or night! On Ubuntu 17.10 with my monitor and graphics card, Flux doesn't work.

**Comment by pdimri** [comment]
I already left the browser. I am using yandex + chrome dark extension , it
works for mobile

**Comment by DatrixTHLK** [comment]
For me f.lux does the job, but doesn't excuse not having a dark mode on browsers themselves. I thank you for the browser and the things being done with it, but please, I do love Stylish/Greasemonkey.

**Comment by adambirds** [comment]
Dark mode would definitely be appreciated.

**Comment by jamesray1** [comment]
No me toos please.

**Comment by ChrisSalisbury** [comment]
+1

**Comment by Monakiel** [comment]
I'm gonna leave Brave in two weeks max if they do not put dark mode.

**Comment by DatrixTHLK** [comment]
Rather than commenting +1, please put a 👍 as a GitHub Reaction @ChrisSalisbury

**Comment by jeppestaerk** [comment]
+1 👍

**Comment by stianmandarin** [comment]
+1

**Comment by larrydewey** [comment]
Any update on this? This has been in the works for over two years... Do you need me to come on and write it for you?

**Comment by A9G-Data-Droid** [comment]
No need to be so salty, Larry. You're on GitHub... Take a look at the code yourself. You can help if you want. You don't need an invitation or to "come on".

If you sort the issues by "Thumbs Up" you can see that this is the #2 most important issue to the community. I think they know.

**Comment by larrydewey** [comment]
@A9G-Data-Droid: Salty? That is not an accurate adjective. I am feeling more antagonistic than anything. As an open source developer, I already understand how it works, but thank you. Sorting by thumbs up does not dictate that it is second in level of importance, it just means it has been voted on, so I would appreciate it if you didn't retort with an assumption which may be incorrect.

**Comment by A9G-Data-Droid** [comment]
Still being salty.

> (Internet slang, derogatory) Indignant or offended due to over-sensitivity, humourlessness, or defeat (implying the person is a crybaby, shedding salty tears); said of interlocutors expressing indignation, or merely disagreement.

Importance is relative to whom. Notice I validated "to the community". The community in this case being people who have voted here, as suggested many times; "Rather than commenting +1, please put a +1 as a GitHub Reaction". This suggestion makes it easy to see, by sorting on thumbs, what issues are important to the community. Why did I have to explain that to you?

Now that you've retorted with an assumption which may be incorrect I suggest you use your "open source developer" skills to back up your words with actions. Get to work.

**Comment by larrydewey** [comment]
A fascinating definition which does not exist in any vetted dictionary, but I appreciate the sentiment. You do not represent the community, you represent yourself. What is important to the community is not what is important to a team.

As far as your attempt to elicit a reaction through your last two sentences, I would state the same thing to you, instead of wasting your time attempting to defend something which needs no defense, put your money where your mouth is. I'll be looking for a pull request from you as well.

**Comment by A9G-Data-Droid** [comment]
I never stated I would. I merely pointed out that you are being rude and you are welcome to do as you said:

> Do you need me to come on and write it for you?

**Comment by larrydewey** [comment]
Nor did I state that I would, perhaps you would benefit from researching the difference between an interrogative and declarative statements. Regardless, I hold no enmity toward you @A9G-Data-Droid, we are both people, and beyond the disconnection which exists because of the internet, I am certain that we would both get along in real life. I will contribute a fix, and I will wait for my pull request to be added to the code base. Thus adding substance to my statements, as you have suggested.

**Comment by DatrixTHLK** [comment]
Don't take this the wrong way but, I'm unsubscribing from here because as much as I like a good notification, I don't really see anything important. Thanks for putting up a fix though larry. :) Hope something happens.

**Comment by pdimri** [comment]
Is night mode too hard to implement from dev perspective? It’s actually most awaited and desired feature from brave team.

**Comment by DatrixTHLK** [comment]
It is second most awaited as far as I've heard. What's the first I wonder?

**Comment by jlawhon** [comment]
Adding a +1 to this.  Working in a semi-lit environment, the start-white is harsh on the eyes.  Options to reduce the glare would be most appreciated.

**Comment by dhniels** [comment]
Tossing my hat in the ring for a dark mode - would be a really nice feature. Seems like a lot of sites nowadays are offering night modes and being able to toggle brave browser interface to night mode along with the website would be nice :)

**Comment by iamaya** [comment]
2 years and still waiting.. + 10 Million. Please implement Dark Mode.

**Comment by A9G-Data-Droid** [comment]
It's still in "planning" stage. It's not even marked "In Progress" yet. This issue has the #1 most comments and the #2 most thumbs up. I would like to see it move to "In Progress" at least.

**Comment by wwwdata** [comment]
As far as I understood from this blog post here: https://brave.com/development-plans-for-upcoming-release/
this ticket is going to be obsolete when the switch to the chromium frontend is done. Is that correct?

**Comment by bradleyrichter** [comment]
A dark toolbar theme for the new Chrome based UI is easy. But I’d like to keep this thread alive for the harder and more important task of “dark mode” which will need to include darkening of all styles in the UI including the white backgrounds of the prefs (settings) pages.

Dark theme (toolbars) + Darkening DOM content + dark browser page content = complete success.


Some of these may still be possible in Brave Muon on the way to the new Brave Chromium.

**Comment by hojsone** [comment]
+1
Please add Dark theme(Mode) so I can completely migrate to Brave, for now I cannot use it much as it is too bright. 
Looking forward to the Dark launch!

**Comment by jonl-percsolutions-com** [comment]
Could we not just ad this extension as a stop gap or for those that don't want a full theme:

https://chrome.google.com/webstore/detail/high-contrast/djcfdncoelnlbldjfhinnjlhdjlikmph

**Comment by A9G-Data-Droid** [comment]
If people could load their own extensions they could do anything they want. I think that is also a feature that is coming soon. Then the browser can be as light or feature rich as you want.

**Comment by bsclifton** [comment]
Labelling with `fixed-with-fork`. The Chromium fork which we've done ([coming later this year - click here for the blog post](https://brave.com/development-plans-for-upcoming-release/)) fixes this problem.  You can take a peek at the different themes that @jamesmudgett has put together here (both a dark UI and a light UI):
https://github.com/brave/brave-chromium-themes

We don't have any binaries available because it's still a work-in-progress, but in case folks would like to try building from source, please check out:
https://github.com/brave/brave-browser

**Comment by MariaCobretti** [comment]
I am blinded

**Comment by Alrikja** [comment]
+1 👍

**Comment by jubbbird** [comment]
I intend to do a primary browser trial just as soon as there's a dark theme.

**Comment by tron-solo** [comment]
+1

**Comment by jtveg** [comment]
Will a dark theme be available for the Android version of Brave?

**Comment by bsclifton** [comment]
@jtveg here's the issue for the Android version in case you wanted to post or give it a thumbs up 😄 
https://github.com/brave/browser-android-tabs/issues/81

**Comment by killernot** [comment]
Why is this not yet implemented?

**Comment by bsclifton** [comment]
@killernot see post above (https://github.com/brave/browser-laptop/issues/271#issuecomment-383998029)

With the current Muon browser (browser-laptop), this can be quite a chore. We'd definitely welcome any contributions helping achieve this goal, but we do already have this working with our new Chromium UI (which should be available in a few months)

**Comment by betegon** [comment]
+1

**Comment by KHarker** [comment]
+1 👍

**Comment by greyl0ck** [comment]
+1 👍

**Comment by lunarthegrey** [comment]
+1 :+1:

**Comment by JonnyRedHed** [comment]
For me its more than just the UI that needs to be dark.  Having support for Stylus is also important.  I make every dark when I can, not black dark but dark deep blue/grey etc, dimmed white text etc.   And when not using stylus and I come across a what BG news article etc I use the chrome extension 'Care your eyes'.

So can we have support for at least Stylus please.  Your own Dark UI will be great, just like when Opera added there dark mode UI.      Also the back end settings pages, extension page etc would be good to also have dark.    

I've no coder but it has been explained to me its tricky to do for Opera and Brave.  


---------------------

And old post below here showing Stylus and Care your Eyes.


For me at the moment to make web pages dark, the best two options are below in my opinion.

'Care your eyes' - chrome extension - can be used in Opera, Chrome, Vivaldi etc. Good for random web pages you want dark. Such as some white background article you land on etc but may never return to that site.

Care your eyes uses a nice dark brown hue for the dark mode which is quite a nice balance of night time or dimmed room colors. It also Remembers top level domains. Dev is working on remembering single pages rather than just top level domain.

*set up: choose white list mode, then just use the 'Enable at this site' to make a page dark and have it remember. Very easy for those random web pages and articles you want dark.

url:
https://chrome.google.com/webstore/search/care%20your%20eyes

--
Stylus - better than Stylish - no data mining etc of your history and habits.
Far fewer permissions than stylish.
https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne?hl=en

Stylus Dark UI - Manage styles page etc with the stylus UI in dark mode.
https://userstyles.org/styles/142097/stylus-gray-matter-chromium-based-or-firefox

**Comment by tconn89** [comment]
👍 
+1 for Dark mode!

**Comment by so-lost** [comment]
+1 👍

**Comment by h3298** [comment]
When you say "Dark UI" there are really two separate issues here: a dark theme for the browser, and a night reader mode for web sites.  While **TLV** and **Dark Reader** both have 5-star ratings in the Chrome store, Dark Reader has a much larger user base.  I have not tested TLV, but did try everything else I could find in the way of night browsing extensions.  The result of those tests is that Dark Reader seems to make the fewest mistakes when inverting bright pages and graphics.  After browsing thousands of sites over several months, I only found a couple dozen which Dark Reader did not handle adequately... and development is constantly progressing.  I would have tested TLV too, if it worked on Firefox like Dark Reader does.  So I would suggest that TLV or Dark Reader should be incorporated as default options in Brave.  Unfortunately, I cannot use Brave on a regular basis without a night mode.

**TLV**
https://chrome.google.com/webstore/detail/tlv-night-mode/jdkclpamgobjdopkemcojdhbfddoplgg

**Dark Reader**
https://chrome.google.com/webstore/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh

**Comment by JonnyRedHed** [comment]
For me they just need to allow Stylus extension and then we can import our own set.  And for those pages I just happen upon that I want to darken just to read an article or what ever, then I use the 'Care your Eyes' Extension.  Its night time mode is a nice dark brown hue/blend.

**Comment by h3298** [comment]
@JonnyRedHed  

> they just need to allow the Stylus extension, then we can import our own set of dark styles 

You can make your own styles with Dark Reader, try it =)

![](https://www.softzone.es/app/uploads/2018/02/Extension-Dark-Reader-Chrome-Firefox.png)

**Comment by JonnyRedHed** [comment]
Brave supports neither at the moment.

**Comment by h3298** [comment]
Half of the population will never use Brave if it does not support extensions, and the company will go out of business

**Comment by eljuno** [comment]
@h3298 please see comment above https://github.com/brave/browser-laptop/issues/271#issuecomment-383998029 for dark theme.

And brave-core will have more extensions support. https://brave.com/development-plans-for-upcoming-release/

**Comment by adambaybutt** [comment]
+1

not going blind is cool...

**Comment by superherointj** [comment]
How do I enable Dark mode in Brave?

**Comment by JonnyRedHed** [comment]
There is no dark mode UI to enable. Not yet at least, thats the whole point of this request thread.

**Comment by millette** [comment]
You enable it with your mind! :-)

**Comment by tomlowenthal** [comment]
We have a dark UI working right now in the Chromium-based Brave refactor! It doesn't cover tasks like changing the color-palettes of sites, but there's also wide extension support which includes Dark Reader or similar tools. We're not going to continue any work on this within the current Muon-based builds. The dark theme should be easy on your eyes soon enough.

**Comment by bsclifton** [comment]
Screenshot of the Dark theme in action:
![screen shot 2018-08-24 at 2 53 36 pm](https://user-images.githubusercontent.com/4733304/44609684-8b1e4e00-a7ad-11e8-9235-2a620b2e059f.png)

**Comment by JonnyRedHed** [comment]
So this is work in progress, to be released at a later time.

If you have followed the Opera browser devs they have been working hard on correcting the tab bar for lots of website icons. Either they are seeing these stats reported by the browser or just working thru a list of the webs most visited websites. I'm guessing from stats reporting in browsers with those users who have it turned on.  They have been better matching the dark mode UI with fav icons,  either replacing the fav icons or working on the edge aliasing effects for them so they stand out better.

I assume Brave devs will also be doing this.

Also with this latest Opera version https://blogs.opera.com/desktop/2018/08/opera-55-0-2994-44-stable-update/  they have migrated over the dark mode for the settings page from the dev browser.  Also the stable channel release also has the update and recovery page in dark mode.  And they are actively working on the webpage right click context menu, and other drop down menus all being dark when the main dark mode UI is turned on.   All looking very good so far.     Opera, Brave and Vivaldi should be among the best browsers having full on dark mode UI's.

Keep up the good work!

**Comment by JonnyRedHed** [comment]
Shouldn't this be left open, for further comment by users.

**Comment by bsclifton** [comment]
@JonnyRedHed it's experimental at the moment, but you can try the new Brave here:
https://github.com/brave/brave-browser/releases

We're working hard on getting this ready for Beta. Let us know what you think 😄

**Comment by JonnyRedHed** [comment]
I couldn't get BraveBrowserDevSetup.exe to install.

**Comment by srirambv** [comment]
@JonnyRedHed  Could you please log an issue in https://github.com/brave/brave-browser/issues and provide as much details as possible so that we can  find out the root cause

**Comment by tomlowenthal** [comment]
Any future work in this direction will be in [brave-browser](https://github.com/brave/brave-browser/issues). Nothing else is going to change in the current Muon-based Brave. The updated Chromium-based Brave will have:
- a dark browser theme available by default
- support for various extensions which change site appearance

Further discussion on this topic should head over to the [community forum](https://community.brave.com/search?q=dark).
